src/Providers/RouteServiceProvider.php

```diff
@@ -70,13 +70,28 @@ protected function configureRateLimiting(): void
      */
     protected function mapWebRoutes(): void
     {
+        Authenticate::redirectUsing(fn (Request $request) => route('login', absolute: false));
```

Comment:
Only do this, if FluxRoutes are being registered, otherwise the login route may not exist.